src/test/java/uk/gov/hmcts/cmc/claimstore/services/ResponseDeadlineCalculatorTest.java

```diff
@@ -25,91 +17,31 @@
     private static final int DAYS_FOR_SERVICE = 5;
     private static final int POSTPONE_BY = 14;
 
-    @Mock
-    private BankHolidaysApi bankHolidaysApi;
-
     private ResponseDeadlineCalculator calculator;
 
     @Before
-    public void setUp() throws IOException {
-        WorkingDayIndicator workingDayIndicator = new WorkingDayIndicator(
-            new PublicHolidaysCollection(bankHolidaysApi)
```

Comment:
Is the bank holidays api still used?

Reply:
yes for `OfferResponseDeadlineCalculator` and other services.